src/gui/fileactivitylistmodel.cpp

```diff
@@ -61,10 +50,12 @@ void FileActivityListModel::startFetchJob()
     QObject::connect(job, &JsonApiJob::jsonReceived,
         this, &FileActivityListModel::activitiesReceived);
 
+    qDebug() << "FETCHING FILE WITH ID " << _objectId;
```

Comment:
can you check if this is needed and if you want to keep it, please use categorized logging like we do in most other places